Medical-Units-MSSC/src/main/java/com/clinics/medicalunits/ui/repositorie/MedicalUnitRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.clinics.medicalunits.ui.repositorie;
+
+import com.clinics.medicalunits.ui.model.MedicalUnit;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface MedicalUnitRepository extends JpaRepository<MedicalUnit, Long> {
+
+	@Query("SELECT m FROM medical_unit m")
```

Comment:
Is that needed ?

Reply:
My mistake, I couldn't remember findAll();